src/app/modules/attribution/features/feature/feature.html

```diff
@@ -1,149 +1,146 @@
-<ng-container class="feature-card" [formGroup]="attribution.form">
-  <ng-container formArrayName="presets">
-    <ng-container [formGroupName]="i">
-      <div class="row">
-        <div class="col-md-12">
-          <div class="feature-group">
-            <div class="content-group">
-              <div class='feature-header'>
-                <label class="content-label feature-label">
-                  <h5>Identifying Tags</h5>
-                </label>
-              </div>
-              <div class="content-body feature-body">
-                <table class="table table-bordered feature-table">
-                  <thead class="feature-table-thead">
-                    <tr class="feature-table-head-row">
-                      <th class="feature-table-main feature-table-th" scope="col">key</th>
-                      <th class="feature-table-main feature-table-th" scope="col">value</th>
-                      <th class="feature-table-delete feature-table-th" scope="col"></th>
-                    </tr>
-                  </thead>
-                  <tbody>
-                    <ng-container formArrayName="primary">
-                      <ng-container *ngFor="let primaryGroup of primaryFormArray.controls; let primaryGroupIndex=index">
-                        <ng-container [formGroupName]="primaryGroupIndex">
-                          <tr class="feature-table-body-row">
-                            <td class="feature-table-main feature-table-td">
-                              <ng-container attribution field="key" [group]="attribution.form" [nestedGroupIndex]="i"
-                                [nestedArrayIndex]="primaryGroupIndex">
-                              </ng-container>
-                            </td>
-                            <td class="feature-table-main feature-table-td">
-                              <ng-container *ngIf="primaryFormArray.at(primaryGroupIndex).get('key') && primaryFormArray.at(primaryGroupIndex).get('key').value && primaryFormArray.at(primaryGroupIndex).get('val')">
-                                <ng-container primary-identifier field="val" [group]="attribution.form"
-                                  [nestedGroupIndex]="i" [nestedArrayIndex]="primaryGroupIndex">
-                                </ng-container>
-                              </ng-container>
-                            </td>
-                            <td class="feature-table-delete feature-table-td">
-                              <button class="btn btn-sm btn-light feature-button feature-table-delete-button"
-                                type="button" (click)="removePrimaryGroup(primaryGroupIndex)">
-                                <i class="fa fa-times"> </i>
-                              </button>
-                            </td>
-                          </tr>
-                        </ng-container>
-                      </ng-container>
+<ng-container [formGroupName]="i">
```

Comment:
none of the new stuff uses this. I might have started working here then stopped.